templates/syncctl/deployment.yaml

```
@@ -104,3 +104,5 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      topologySpreadConstraints:
+        {{- tpl .Values.syncctl.topologySpreadConstraints . | nindent 8 }}
```

What do you think about doing this in order to support both a template and a list?
```suggestion
      {{- with .Values.syncctl.topologySpreadConstraints }}
      topologySpreadConstraints:
        {{- if kindIs "string" . }}
        {{- tpl . $ | nindent 8 }}
        {{- else }}
        {{- toYaml . | nindent 8 }}
        {{- end }}
      {{- end }}
```